apps/corporatebsgen/corporate_bs_gen.star

```diff
@@ -0,0 +1,48 @@
+""")
+
+def main():
+    phrase = cache.get("corporate_bs")
+    if phrase != None:
+        print("Cache Hit!")
+    else:
+        print("Cache Miss!")
+        rep = http.get(CORPORATE_BS)
```

Comment:
You can now pass `ttl_seconds` straight to `http.get()` to have the http module handle caching for you. This simplifies the code a bit, and also future proofs against (possible) coming changes to how cache module is scoped.